### PR TITLE
TFLM: Exclude micro speech example

### PR DESCRIPTION
**Pull request by mansnils**
Excluding micro speech example in cortex_m_generic_makefile.inc to
prevent warnings to be cleared.

Fix for: https://github.com/tensorflow/tensorflow/issues/47589
It also progress towards: https://github.com/tensorflow/tensorflow/issues/47070

### CODE REVIEW

File: tensorflow/lite/micro/tools/make/targets/cortex_m_generic_makefile.inc [hunk 1]
```diff
@@ -111,14 +111,21 @@ else
   $(error "TOOLCHAIN=$(TOOLCHAIN) is not supported.")
 endif
 
+# TODO: fix CMSIS-NN related warnings.
```
Comment (advaitjain) [inline tensorflow/lite/micro/tools/make/targets/cortex_m_generic_makefile.inc:114]: please add a github issue to the TODO.

e.g. TODO(#1234):

**Review by advaitjain — CHANGES_REQUESTED** [review]

File: tensorflow/lite/micro/tools/make/targets/cortex_m_generic_makefile.inc (hunk 1 above)
Comment (mansnils) [reply tensorflow/lite/micro/tools/make/targets/cortex_m_generic_makefile.inc:114]: Done

**Review by advaitjain — APPROVED** [review]

**Comment by rthadur** [comment]
@mansnils can you please resolve conflicts

**Comment by mansnils** [comment]
@rthadur conflict is resolved

**Review by advaitjain — APPROVED** [review]

**Pull request merged**